src/Components/Welcome/Welcome.css

```diff
@@ -0,0 +1,20 @@
+
+.popup {
+  display: inline-block;
+  visibility: hidden;
```

Comment:
For future reference, there's an HTML/XML tag called dialog for popups. that might be a better choice here.